src/components/locator.ts

```diff
@@ -93,7 +93,7 @@ export class Locator {
         }
     }
 
-    syncTeX(args?: {line: number, filePath: string}, forcedViewer: string = 'auto') {
+    syncTeX(args?: {line: number, filePath: string}, forcedViewer: string = 'auto', pdfFile?: string) {
```

Comment:
This function is too long. It needs refactoring.

Reply:
I agree that the `syncTeX` function is quite lengthy but its logic is quite clear and many others are also lengthy.
I have not changed much in this function actually. So I think I am going to leave it like it.

Some day, we should take some time to do a bit a refactoring.

Reply:
(related to jlelong's comment: #1475)